components/batch-upload/SnapshotUpload.tsx

```diff
@@ -90,7 +90,10 @@ export const SnapshotUpload: React.FC<SnapshotUploadProps> = ({
           const data: SnapshotAddressInput[] = (
             results.data as SnapshotAddressInput[]
           )
-            .map((a) => ({ ...a, address: (a.address || "").trim() }))
+            .map(({ address, maxClaimable }) => ({
+              address: (address || "").trim(),
+              quantity: maxClaimable ? (maxClaimable || "").trim() : "0",
```

Comment:
are we sure we want to set "0" as the default instead of just not having the key?

Reply:
yes, "0" is what defaults to the one set in the claim phases, it's already the default